game/marvin.cpp

```diff
@@ -237,6 +243,67 @@ bool Marvin::exec(std::string_view v) {
         }
       return true;
       }
+    case C_CheatGod: {
+      Gothic::inst().setGodMode();
+      auto& fnt = Resources::font();
+      if(Gothic::inst().isGodMode())
+        Gothic::inst().onPrintScreen("Godmode on",2,4,1,fnt);
+      else
+        Gothic::inst().onPrintScreen("Godmode off",2,4,1,fnt);
+      return true;
+      }
+    case C_Kill: {
+      Npc* player = Gothic::inst().player();
+      if(player==nullptr || player->target()==nullptr)
+        return false;
+      auto target = player->target();
+      target->changeAttribute(ATR_HITPOINTS,-target->attribute(ATR_HITPOINTSMAX),false);
+      return true;
+      }
+    case C_AiGoTo: {
+      World* world  = Gothic::inst().world();
+      Npc*   player = Gothic::inst().player();
+      if(world==nullptr || player==nullptr || !player->setInteraction(nullptr))
+        return false;
+      auto wpoint = world->findPoint(ret.argv[0]);
+      if(wpoint==nullptr)
+        return false;
+      player->aiPush(AiQueue::aiGoToPoint(*wpoint));
+      return true;
+      }
+    case C_GoToPos: {
+      Npc* player = Gothic::inst().player();
+      if(player==nullptr || !player->setInteraction(nullptr))
+        return false;
+      float x,y,z;
+      try {
+        x = std::stof(ret.argv[0].data());
```

Comment:
Usage of `stof` with `string_view` is not correct, since string_view is not zero-terminated.
Need to use `std::from_chars`